skyportal/handlers/api/news_feed.py

```diff
@@ -104,6 +114,39 @@ def fetch_newest(model):
                 for c in comments
             ]
         )
+        news_feed_items.extend(
+            [
+                {
+                    "type": "classification",
+                    "time": c.created_at,
+                    "message": f"New classification for {c.obj_id} added by {c.author.username}: {c.classification}",
+                    "source_id": c.obj_id,
+                }
+                for c in classifications
+            ]
+        )
+        news_feed_items.extend(
+            [
+                {
+                    "type": "spectrum",
+                    "time": s.created_at,
+                    "message": f"New spectrum taken with {s.instrument.name} uploaded for {s.obj_id}",
+                    "source_id": s.obj_id,
+                }
+                for s in spectra
+            ]
+        )
+        news_feed_items.extend(
+            [
+                {
+                    "type": "photometry",
+                    "time": p.created_at,
+                    "message": f"Follow-up photometry taken for {p.obj_id}",
```

Comment:
Describe who uploaded the photometry and what instrument it was taken with,

```suggestion
                    "message": f"{p.owner.first_name} {p.owner.last_name} uploaded a {p.instrument.name} spectrum for {p.obj_id}",
```

Reply:
Done